### DISCUSSION TITLE
Can a worker be in the `tagged` and `untagged` pools

### DISCUSSION THREAD

**Discussion by robbadler** [concourse/concourse#8952] [Help & Support] [answered]
I have two sets of worker pools, one of which uses a static route in order to have it's IP allow-listed on some downstream services, and one which uses random ephemeral IPs. Otherwise, these workers are the same. Ideally, "untagged" work for me would be scheduled across both of these pools, and any work which requires access to the downstream service could then use the appropriate tag. 

From reading the documents, the only was I can see to do this would be to tag every worker `generic` and then the static workers with `static`, as well as tagging every `resource`, `get`, `put`, and `task` with `tags: [generic]`.

Is there any other way to get the behavior I seek?

**Comment by taylorsilva** [comment, marked as answer]
You are correct in your reading of the docs. Your proposal would be the only way to achieve your desired behaviour.

**Comment by robbadler** [comment]
Thanks @taylorsilva .

I'll look at raising an issue or feature request around this, and see if there is an elegant or easier solution (both in code, or in pipeline-definition).
